src/components/VmStatus/getVmStatus.js

```diff
@@ -99,47 +109,7 @@ const isVmError = vm => {
   return NOT_HANDLED;
 };
 
-const isBeingImported = (vm, importerPods) => {
-  if (importerPods && importerPods.length > 0 && !get(vm, 'status.created', false)) {
-    const importerPodsStatuses = importerPods.map(pod => {
-      if (!isSchedulable(pod)) {
-        return {
-          status: VM_STATUS_IMPORT_ERROR,
-          message: 'Importer pod scheduling failed.',
-          pod,
-        };
-      }
-
-      const failingContainer = getFailingContainerStatus(pod);
-      if (failingContainer) {
-        return {
-          status: VM_STATUS_IMPORT_ERROR,
-          message: getContainerStatusReason(failingContainer),
-          pod,
-        };
-      }
-      return {
-        status: VM_STATUS_IMPORTING,
-        message: getNotRedyConditionMessage(pod),
-        pod,
-      };
-    });
-    const importErrorStatus = importerPodsStatuses.find(status => status.status === VM_STATUS_IMPORT_ERROR);
-    const message = importerPodsStatuses
```

Comment:
can we have this functionality preserved? I think it might be useful

Reply:
added messages back. See screenshots

![screenshot from 2019-02-27 11-31-11](https://user-images.githubusercontent.com/2078045/53484377-0118b000-3a84-11e9-99bc-cb6da5be6d0e.png)
![screenshot from 2019-02-27 11-33-15](https://user-images.githubusercontent.com/2078045/53484384-02e27380-3a84-11e9-9e1a-8d04c4c89c7f.png)

Reply:
I wonder if having "Anotherdisk import Failed" and "rootdisk import failed" Would be a better way of presenting these? A little more readable. 
Nitpick on design, but the tooltip is missing the arrow and the spacing seems odd for the first example.

Reply:
> I wonder if having "Anotherdisk import Failed" and "rootdisk import failed" Would be a better way of presenting these? A little more readable.
+1, messages changed to \<disk_name\> import failed

> Nitpick on design, but the tooltip is missing the arrow and the spacing seems odd for the first example.

Spacing fixed, but the tooltip is actually link title and it is in the same style as other link titles

![screenshot from 2019-03-01 14-26-26](https://user-images.githubusercontent.com/2078045/53641040-1a5d6000-3c2e-11e9-85e5-758486e59f4b.png)

Reply:
it was actually using pf-react tooltip in the past, but it was having problems with breaking long urls

Reply:
Do you mean just using `title="text"` on the link? If so, we should really explore a different approach. The title attribute is not typically exposed in an accessible manner so it wouldn't be available to all users. I would think that a tooltip where a user could click to get further info would be most useful here.

Reply:
@matthewcarleton yes i meant just `title="text"`. Thanks for explanation,  I will explore the tooltip option.

Reply:
Followup created https://github.com/kubevirt/web-ui-components/issues/254

Reply:
great, thanks!